packages/theme/src/style.ts

```diff
@@ -0,0 +1,26 @@
+import type { StyledProps } from 'styled-components'
+
+import breakpoints from './breakpoints'
+
+export interface StyleProps {
+  /**
+   * CSS properties.
+   */
+  sx?: StyledProps<any> // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
This `sx` name is always throwing me off. I know it's used by MUI, but could we have a human-readable name instead?  Or at least a description of what it means everywhere it's used.

Reply:
So we can't use `style` since that's reserved. If we call it `inlineStyle`, it's incorrect since style-components create CSS class for it. If we want to call it something else, I'm afraid it's going to be a really long and unwieldy prop name

Given we have TypeScript types, documentation, and Storybook examples, I think we should be ok unless you have another suggestion

Reply:
Not sure, but I get confused with `sx`, especially if I'm passing an `xs` breakpoint; it feels very cryptic to me. I had to google what `sx` was.

What about `styles`(note the plural)? Is that reserved as well? If we are settled on `sx` that's fine, but I do think it'll be good to add what it means in every instance.

Reply:
Ok let's check with the team on `styles`

Reply:
Actually, we are already using `sx` prop on the frontend so if we changed it, it would be a breaking change (I prefer to support 1 prop instead of 2)

Reply:
Makes sense. Can we document it somewhere?

Reply:
The types should be documented when we import the component. The prop is also documented in the [README](https://github.com/little-spoon-dev/design-system/tree/%40littlespoon/box%401.3.0/packages/box)

Reply:
We're going to sync in RFC on the prop name but we will be merging this first